### PR TITLE
Hotfix/fix unit test

### PR DESCRIPTION
**Pull request by neveldo**
@Indigo744 , maybe you are still working on it, but here is some little fixes on two unit tests.

### CODE REVIEW

File: test/test-options.js [hunk 1]
```diff
@@ -95,21 +95,19 @@ $(function() {
...
-        $('<div/>').addClass(container_class).appendTo('.container');
+        $('<div/>').addClass(tooltip_class).appendTo('.container');
         
         $(".mapcontainer").mapael({
             map: { 
                 name: "france_departments",
                 tooltip: {
                     cssClass: tooltip_class,
-                    css: additional_prop,
-                    target: $("." + container_class)
```
Comment (neveldo) [inline test/test-options.js]: Target option aims to set a target for the link, like '_blank', not a target container

File: test/test-options.js [hunk 2]
```diff
@@ -95,21 +95,19 @@ $(function() {
     
     test("Tooltip options", function(assert) {
         var tooltip_async_done = assert.async(CST_NB_OF_FRANCE_DPTMT);
-        var container_class = "CONTAINER_CLASSNAME";
         var tooltip_class = "TOOLTIP_CLASSNAME";
         var additional_prop = {
-            "border-left": "5"
+            "border-left": "5px solid rgb(0, 255, 0)"
```
Comment (neveldo) [inline test/test-options.js]: The border style have to be fully filled otherwise it breaks the check.

File: test/test-options.js [hunk 3]
```diff
@@ -66,13 +66,13 @@ $(function() {
         assert.throws(function(){
             $(".mapcontainer").mapael({
                 map: { 
-                    name: "not_existing_map",
+                    name: "france_departments",
```
Comment (neveldo) [inline test/test-options.js]: I think we should only check for an non existing cssClass in this test

Comment (Indigo744) [inline test/test-options.js]: And you are right! This was a mistake :-)
Too much copy/paste...

File: test/test-options.js (hunk 2 above)
Comment (Indigo744) [inline test/test-options.js]: Yes, I was having trouble with this test. I left it "as is" to be corrected later ;-)

File: test/test-options.js (hunk 1 above)
Comment (Indigo744) [inline test/test-options.js]: Oh ok! You see, I didn't understood at all the purpose of this option. Maybe we should rephrase the help message?

Comment (neveldo) [inline test/test-options.js]: My bad, I am also confused about this option. It was indeed used to set a custom container for the tooltip div, but  we have deleted it some weeks ago as it is not useful anymore : https://github.com/neveldo/jQuery-Mapael/commit/f1e8758b91504f399392e4af390ed020ad935bdf .

**Pull request merged**